docs/sessions.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+---
+
+State and Sessions in Suave
+===========================
+
+While stateless web applications are all the rage, there are still times where state needs to be preserved, either across requests in a session, or simply among `WebPart`s as the request/response pipeline is composed.  In these cases, Suave has you covered!  We'll look at three aspects of session cookies first, then how we can construct state among `WebPart`s.
```

Comment:
But even if you persist across sessions (in the cookie storage) the web app **is** stateless in the meaning intended when people speak of stateless web apps. Similarly, a pipeline as part of processing the request always has some state. I think this may confound the matter more than it helps.